.eslintrc

```diff
@@ -0,0 +1,55 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+      "warnOnUnsupportedTypeScriptVersion": false,
+      "ecmaFeatures": {},
+      "ecmaVersion": 2018,
+      "sourceType": "module",
+      "project": "./tsconfig.json"
+    },
+    "settings": {},
+    "globals": {},
+    "plugins": ["@typescript-eslint", "jasmine"],
+    "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+    "env": {
+      "jasmine": true,
+      "browser": true,
+      "es6": true
+    },
+
+    "rules": {
```

Comment:
`tslint.json` had the rules in alphabetical order. Would be nice, but not a big deal.